services/health/Cargo.toml

```diff
@@ -12,4 +12,6 @@ splinter = { path = "../../libsplinter" }
 [features]
 default = []
 
+stable = ["default"]
```

Comment:
Do we know how this sort of arrangement will work with `--no-default-features --features stable`?

Reply:
Works fine in my tests